.github/workflows/github.yml

```diff
@@ -276,23 +335,26 @@ jobs:
         mv "$OUTPUT_DIRECTORY/bundle/release/vcmi-release.aab" "${{github.workspace}}/out/build/${{matrix.preset}}/${{ env.VCMI_PACKAGE_FILE_NAME }}.aab"
 
     - name: Upload Artifact
+      id: upload_artifact
       uses: actions/upload-artifact@v4
       with:
         name: ${{ env.VCMI_PACKAGE_FILE_NAME }} - ${{ matrix.platform }}
-        compression-level: 0
+        compression-level: 9
```

Comment:
Why? Idea was to avoid wasting time on compression of already compressed data. Or it can actually improve zipping/artifact upload time?